src/components/dropdowns/BusManagement.js

```diff
@@ -13,25 +14,38 @@ const BusManagement = ({ row }) => {
     });
   };
   const selectRef = useRef();
-  const handleDelete = async (busId) => {
-    setloading(true);
-    try {
-      await axios.delete(`/buses/${busId}`);
-      toast('Bus deleted successfully', { type: 'success' });
-      setloading(false);
-      selectRef.current.innerHTML = `<b className="deleted mx-auto font-bold">Deleted</>`;
-    } catch (error) {
-      toast('Error occured', { type: 'error' });
-      setloading(false);
+  const handleDelete = async (busInfo) => {
+    const results = await Swal.fire({
```

Comment:
You can create a custom hook to handle all the delete method and call it inside the handleDelete(); see implementation below.

https://reactjs.org/docs/hooks-custom.html